openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -706,6 +724,16 @@ def populate_scene_placeholders(
 
                 placeholder.set_finished()
 
+            # Trigger on_depth_processed event
+            self.events.emit(
+                topic="on_depth_processed",
```

Comment:
```suggestion
                topic="depth_processed",
```

Reply:
Should we make it "template.finished" and "template.depth_processed"?

Reply:
Tweaked it with https://github.com/ynput/OpenPype/pull/5830/commits/d00d2d2d484ccbc0e83786c2385cea032e9ff21c let me know what you think.